pldoc/static/sass/_lib.scss

```diff
@@ -13,3 +13,5 @@
 } @else {
     @import '../../../bower_components/bi-app-sass/bi-app/bi-app-ltr';
 }
+
+@import 'syntax';
```

Comment:
Ahh, its specific to the UXPL Doc Site needs (in this case, to show code). Its a CSS add-on for a Jekyll plugin called Pygments (see http://jekyllrb.com/docs/templates/#stylesheets-for-syntax-highlighting and the entire section above).